src/components/modals/DownloadParticipantsModal.tsx

```diff
@@ -144,14 +145,16 @@ export default function DownloadParticipantsModal({
       visible={visible}
       onCancel={onCancel}
       onOk={download}
-      okText="Download CSV"
+      okText={t`Download CSV`}
       okButtonProps={{ type: 'primary' }}
-      cancelText="Close"
+      cancelText={t`Close`}
       confirmLoading={loading}
       centered
     >
       <div>
-        <h4>Download CSV of {tokenSymbol || 'token'} holders</h4>
+        <h4>
+          <Trans>Download CSV of {tokenSymbol || 'token'} holders</Trans>
```

Comment:
**suggestion**:

```suggestion
          <Trans>Download CSV of {tokenSymbol || t`token`} holders</Trans>
```

Although, I don't think this is the best way to handle this. Some languages might have a different word than `token` depending on the context...

Maybe we can leave this as-is.